cw-orch-daemon/src/builder.rs

```diff
@@ -53,6 +55,12 @@ impl DaemonAsyncBuilder {
         self
     }
 
+    /// Specifies wether authz should be used with this daemon
+    pub fn with_authz(&mut self, granter: impl ToString) -> &mut Self {
```

Comment:
Would prefer `with_authz_granter` for clarity. `with_authz` is ambiguous as to what the functionality would be.

Reply:
Also - the other builders don't use the `with_` prefix, is there a reason for choosing to use it for authz specifically?

Reply:
Yes, changing